backend/config.js

```diff
@@ -33,6 +33,10 @@ module.exports = {
       url: process.env.API_ENDPOINT_URL || 'http://localhost:8080',
       timeoutSeconds: toInt(process.env.API_ENDPOINT_TIMEOUT_SECONDS, 30),
     },
+    manageusers: {
+      url: process.env.API_ENDPOINT_URL || 'http://localhost:9091',
```

Comment:
this doesn't look right - needs to be a different env variable from `API_ENDPOINT_URL` as that is used by prison api

Reply:
yes will update